config/jobs/kubernetes/sig-node/dynamic-resource-allocation.yaml

```diff
@@ -136,7 +142,7 @@ periodics:
         - --deployment=node
         - --env=KUBE_SSH_USER=core
         - --gcp-zone=us-west1-b
-        - '--node-test-args=--feature-gates="DynamicResourceAllocation=true" --service-feature-gates="DynamicResourceAllocation=true" --runtime-config=api/alpha=true,api/beta=true --container-runtime-endpoint=unix:///var/run/crio/crio.sock --container-runtime-process-name=/usr/local/bin/crio --container-runtime-pid-file= --kubelet-flags="--cgroup-driver=systemd --cgroups-per-qos=true --cgroup-root=/ --runtime-cgroups=/system.slice/crio.service --kubelet-cgroups=/system.slice/kubelet.service" --extra-log="{\"name\": \"crio.log\", \"journalctl\": [\"-u\", \"crio\"]}"'
```

Comment:
I thought DRA feature gates were off in beta due to API skew.

Reply:
Ahem - yes. Thanks for reminding me! :hot_face: 

FWIW, I plan to introduce "-canary" variants for all of the jobs to make it safe to change the jobs: first try a change in the canary jobs, then change the real ones accordingly and ideally automatically.

Reply:
I got stuck testing these jobs locally (see https://kubernetes.slack.com/archives/C09QZ4DQB/p1734442263607439). For now let's merge and hope that it doesn't break anything.

I've added `--feature-gates` and `--service-feature-gates` back, but without the double quotes (not necessary).